DocumentedExamples/Binning_transformation_from_depth_to_potential_density.ipynb

```diff
@@ -18,7 +18,7 @@
     "2. `xgcm` conservative binning (define edge of bins, but vertically interpolates so looks smoother than `xhistogram`)\n",
```

Comment:
<p>There's a lot of warning output here that ends in a <code>KeyboardInterrupt</code>. It would be nice not to have that output in repo version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>